Exceptions/Exceptions/Program.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace Exceptions
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+
+            //Basic Exception handling
+            try
+            {
+                Console.WriteLine("Enter first number: ");
+                int num1 = GetUserInput();
+
+                Console.WriteLine("Enter second number: ");
+                int num2 = GetUserInput();
+
+                if (num2 == 0)
+                {
+                    throw new DivideByZeroException("Cannot divide by zero");
+                }
+                int result = num1 / num2;
+                Console.WriteLine($"Result of division is {result}");
+            }
+            catch (DivideByZeroException ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"Error occurred: {ex.Message}");
+            }
+
+
+
+            //Custom Exception handling
+            try
+            {
+                Console.WriteLine("Enter a value: ");
+                int value = GetUserInput();
+
+                if (value < 0)
+                {
+                    throw new InvalidInputException("Negative value not allowed");
```

Comment:
@VikramSunke  change the message according to the condition , there we are checking "less than zero" the message either opposite to the condition or condition should be relevant to the requirement.